### ISSUE TITLE
Feature - Remove posts from a channel

### ISSUE THREAD

**Issue by jackjamieson2** [jackjamieson2/yarns-microsub-server#17]
Thinking the best implementation would be to set the post contents to null. This way the post will still be saved so will not be re-fetched by mistake.

**Comment by bekoli** [comment]
Probably related: I got a stuck post that reappears all the time due to a date mistake on publish. Deleting doesn't help.

![Screenshot_2020-01-18-18-19-38](https://user-images.githubusercontent.com/5016428/72979211-24ed8e80-3dd8-11ea-8aac-ccb84d9e1e31.png)

**Comment by jackjamieson2** [comment]
Thanks @bekoli, definitely related. The way it's set up now, Yarns will continually re-fetch that post since the date is new.

1. Need to implement a way to delete posts so they won't be re-fetched.
2. Thinking about how to handle posts with future dates... Probably best to leave as-is, even though it messes up the feed order.  Alternately, if the date is in the future, Yarns could set the feed item's `published` field to the time of fetching. I don't like  the idea of modifying this information (essentially falsifying it) but it might make sense in this case because the publication date is malformed to begin with.  Third option is to refuse to fetch posts that are newer than the present date.   Compromises abound

**Comment by bekoli** [comment]
I see. That's a tricky one indeed. The third option is probably a bad idea. Half of the time there is no date that can be parsed anyway.

**Comment by jackjamieson2** [comment]
Agreed.  Anyway, this is good motivation to at least (finally) implement deleting timeline items.  I'll move that to the top of my list, and hopefully get to it soon. Been a bit difficult to be quick with updates recently
